src/app/core/data/base-response-parsing.service.ts

```diff
@@ -145,4 +149,22 @@ export abstract class BaseResponseParsingService {
     }
     return obj[keys[0]];
   }
+
+  /* TODO remove when REST response for epersons is fixed */
```

Comment:
Thanks for the TODO.  Do we have a REST API bug ticket that this should/could refer to?  I'd like to ensure this gets logged (or has been logged) as a REST API bug.

Reply:
This can be fixed after there is a solution for [DS-4006](https://jira.duraspace.org/browse/DS-4006)